.gitignore

```diff
@@ -21,3 +21,4 @@ console/atest-desktop/atest.exe
 console/atest-desktop/coverage
 atest-store-git
 .db
+.marscode/
```

Comment:
add new blank line

Reply:
Usually, we put a new line at the end of each file. You can search the reason in Google.